cluster.c

```diff
@@ -823,15 +879,17 @@ RRStatus ShardGroupParse(RedisModuleCtx *ctx, RedisModuleString **argv, int argc
 
     id = RedisModule_StringPtrLen(argv[0], &len);
     if (len > RAFT_DBID_LEN) {
-        RedisModule_ReplyWithError(ctx, "ERR: shardgrup id is too long");
+        RedisModule_ReplyWithError(ctx, "ERR: invalid shard group message (shard group id length)");
```

Comment:
My understanding is some clients strip ERR part before passing the error message. In this case, error message will start with ":". Maybe we shouldn't do this? 

```suggestion
        RedisModule_ReplyWithError(ctx, "ERR invalid shard group message (shard group id length)");
```

Reply:
yes, agreed, have had to change that before, keep forgetting